wazo_calld/plugins/dial_mobile/services.py

```diff
@@ -351,6 +352,7 @@ def send_push_notification(
             'video': video_enabled,
             'ring_timeout': ring_timeout,
             'sip_call_id': sip_call_id,
+            'creation_time': creation_time,
```

Comment:
I find the key name `creation_time` not very explicit, did you discuss it already?

Reply:
No. At first it was to reflect the `creation_time` field in calld call events. But I agree it could be renamed to avoid confusion with the actual call `creation_time`.

If we were to be explicit and accurate, though, it would be something like `push_mobile_time`? It would have to be distinguished from the `notification_time` generated by wazo-webhookd.